packages/zoe/src/contractSupport/priceAuthority.js

```diff
@@ -5,10 +5,22 @@ import { Far } from '@endo/marshal';
 import { assert, details as X } from '@agoric/assert';
 import { makePromiseKit } from '@endo/promise-kit';
 import { AmountMath } from '@agoric/ertp';
+import { Nat } from '@agoric/nat';
 import { makeNotifier } from '@agoric/notifier';
 
 import '../../exported.js';
 
+/**
+ * Create a "unit" amount, which is 10**decimalPlaces for a fungible brand.
+ *
+ * @param {Brand<'nat'>} brand
+ */
+export const makeUnitAmount = async brand => {
```

Comment:
This helper seems good, but are (and were) we abusing `getDisplayInfo` ? 
https://github.com/Agoric/agoric-sdk/blob/9a43d45aa6421c43a10acd34a1b898dd24b2dfb5/packages/ERTP/src/types.js#L89-L91

Reply:
I wondered about this too. I think it's a justifiable exception, but it does merit an explicit justification comment.

I think I'd feel better if it were only defined locally where it's being used rather than being exported as if it were encouraged for general use.